src/game_battlealgorithm.cpp

```diff
@@ -931,7 +931,7 @@ bool Game_BattleAlgorithm::Normal::Execute() {
 		}
 		if (GetTarget()->IsDefending()) {
 			if (GetTarget()->HasStrongDefense()) {
-				effect /= 3;
+				effect /= 4;
```

Comment:
didn't cherry mention somewhere that the value is 3?

Reply:
The value is 4.

I made physical skill with 0 str, 0 mag, 0 var, base damage 20.

When it hits Alex:
* Normal 20 dmg
* Defending 10 dmg
* Defending with Super 5 dmg